salalib/tigerp.cpp

```diff
@@ -57,9 +57,9 @@ void TigerMap::parse(const std::vector<string>& fileset, Communicator *comm)
             // grab major code:
             std::string code = line.substr(55,2);
             if (code[0] == 'A' || code[0] == 'B') {
-               size_t index = searchindex(code);
-               if (index == paftl::npos) {
-                  index = add(code,TigerCategory(),paftl::ADD_HERE);
+               auto iter = find(code);

```

Comment:
The find is unnecesary - just use an `insert` and use the first item of the return value, it's either the new entry or an existing one.